Automattic/wp-calypso
# Trail Map: features for comparison grid

Related to #

## Proposed Changes

* Adds trail map features for the comparison grid.

![localhost_64175_iframe html_args= id=comparisongrid--trail-map-copy-and-structure viewMode=story](https://github.com/Automattic/wp-calypso/assets/5436027/9dc583e3-4c62-43af-8c57-675d637b820c)




## Testing Instructions



* Run `yarn workspace @automattic/plans-grid-next storybook`.
* Check the Trail Map stories for the Comparison Grid.
* Confirm that the `Trail Map Control` and `Trail Map Structure` comparison grids are the same as production.
* Confirm that the `Trail Map Copy` and `Trail Map Copy And Structure` comparison grid contain ALL features from the features grid, with the following exceptions:
  * `WordPress.com subdomain` should be hidden.
  * `All free themes` should be hidden.
  * `Limited stats` should be hidden.
  * `All free themes`, `20 premium themes` and `48 premium themes` should be replaced with the `Premium themes` feature from control.
<img width="1599" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/49578112-011e-47c8-a003-d924e1a04047">

  * `Ad-supported experience` should be replaced with `Ad-free experience`.
  * The `Accept payments...` features should be replaced with two feature showing the different % values for each of the plans - this is the same as control.
<img width="1596" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/45963193-3a94-461f-8187-567bb965c0d2">
 

## Pre-merge Checklist
